### ISSUE TITLE
“Popular subgames” and “popular mods” is outdated and unfair

### ISSUE THREAD

**Issue by Wuzzy2** [minetest/minetest.github.io#107]
These sections are pretty bad.

Especially because they become outdated so quickly.

Only 4 subgames + 1 fake subgame are showcased, the selection seems arbitrary. Tutorial isn't even a game. Dreambuilder is not a subgame, but a mod.
Subgames should really be selected by quality, not (perceived) popularity. Popularity is a terrible measurement.

Mods are even worse. This section is horribly outdated and will probably always be. Many mods are so old, they really shouldn't be used anymore.

There is no easy fix. Just trying to keep the sections updated by hand via PRs is a bad solution as PRs are usually extremely time-consuming, and we shouldn't waste so much time whenever a high-quality mod pops up.
I don't have a good idea what to do with mods, if you have one, please let me know!

For subgames, using PRs *might* *kinda* work as there are only a very few subgames of decent quality. Still, it's not really a long-term solution.

But the inclusion criteria for subgames must change. Popularity is completely irrelevant if you only play in singleplayer. What matters is the quality.

I propose to select subgames which are:
* Showing Minetest in a good light
* Enjoyable
* Stable
* Polished (to some degree)
* Working in latest stable Minetest
* Completed, or if WIP, the WIP state is already so good and stable you can play it normally already without having the impression of something lacking
* Passing the Six Hour Test (it must not run out of new content to explore before 6 hours of playing) (non-sandbox games are excempt)
* Having a crafting guide (unless they don't need one)

Also, the subgames/mods page should be re-evaluated after each MT release, in order to scrap everything which has ceased to function.

**Comment by rubenwardy** [comment]
> Especially because they become outdated so quickly.

BFD and Carbone are no longer in development, and are _quite_ old.

> Dreambuilder is not a subgame, but a mod.

It was a subgame, but is now a modpack. It probably shouldn't be featured though, as it's only used on VanessaE's servers. It's better to highlight the mods inside it if they're popular.

>  Subgames should really be selected by quality, not (perceived) popularity. Popularity is a terrible measurement.

I kinda agree with this. Popularity is a crude way to measure quality.

> [bullet points]

I agree with all of these excepted

> Having a crafting guide (unless they don't need one)

This should be encouraged, and a definite bonus, but not required if the game is good otherwise

> There is no easy fix. Just trying to keep the sections updated by hand via PRs is a bad solution as PRs are usually extremely time-consuming, and we shouldn't waste so much time whenever a high-quality mod pops up.

This is really a temporary solution until we have a mod store again which can do this for us.

**Comment by Wuzzy2** [comment]
> > Having a crafting guide (unless they don't need one)

> This should be encouraged, and a definite bonus, but not required if the game is good otherwise

I strongly disagree. Without a crafting guide (or a similar mechanism), the game is **unplayable**. If you don't know the possible crafts, you can't even make a pickaxe. And nobody can remember hundreds of crafting recipes. I do not want to be something like this promoted on the homepage. We have to draw a quality line somewhere, and it is there.

And sorry, there is absolutely no excuse to not have a crafting guide anymore. There are high-quality crafting guide mods readily available, so adding a crafting guide to a subgame is now very easy, it just needs to be done. And most subgames (https://wiki.minetest.net/Crafting_guide) have a crafting guide anyway, so this rule won't even filter out that much.
Long story short: Just add a freaking crafting guide, it isn't hard! Either that, or make your subgame so, that a crafting guide is not needed.

**Comment by Wuzzy2** [comment]
Okay, let's start with the easy part: Remove all broken, low quality or obsolete mods:

* Better HUD (and hunger): known critical bugs, not maintained anymore
* Minecraft-like item drop/pick-up: Old, and there are probably better `item_drop` mods out there
* Craft Guide, Inventory++, Crafting: Extremely old and long superceded by more modern mods
* Weather: Extremely old, we have much better weather mods to offer now
* Snow Biomes: Obsolete
* Nether: As much as I hate to say it, but this isn't really of high quality

Subgames:

* Move Dreambuilder to mods

Texture packs:

* Remove Summerfields. It is (sadly) under a non-free license. Can we at least agree to not promote proprietary *anything* on the *very homepage of Minetest*? ;-)


Agree with my lists? Did I miss something?

**Comment by SmallJoker** [comment]
Just peeking in for a short server list analysis, in case that's helpful to find mods that are used much.
The mods are sorted by the amount of players playing with the specific mod on their server. The percent number is the occurrence of the mod compared with all players.
```
Servers: 120
Clients: 364
Mods:    1474
Fetched mods
                  worldedit 79% (290)
                   3d_armor 71% (260)
                  protector 68% (249)
                       mobs 67% (247)
                      xban2 62% (227)
          unified_inventory 60% (220)
                 itemframes 57% (211)
                     killme 54% (199)
                  wieldview 54% (197)
                    shields 53% (193)
                mobs_animal 50% (184)
                      areas 50% (182)
            building_blocks 48% (178)
                      inbox 48% (177)
                  homedecor 48% (177)
        homedecor_3d_extras 48% (176)
                  signs_lib 47% (174)
               plasmascreen 46% (168)
                     lrfurn 44% (162)
                  fake_fire 44% (162)
                   computer 44% (162)
                     chains 44% (161)
                   lavalamp 43% (157)
                  travelnet 39% (145)
               mobs_monster 37% (135)
                 moreblocks 35% (131)
                    hudbars 35% (129)
                   hbhunger 35% (128)
                        irc 32% (120)
                 city_block 32% (118)
             3d_armor_stand 30% (111)
                    wield3d 29% (107)
           teleport_request 29% (107)
               irc_commands 29% (106)
                    u_skins 28% (105)
                unifieddyes 28% (103)
                3d_armor_ui 27% (99)
                  bakedclay 26% (98)
                    hbarmor 26% (97)
                    afkkick 25% (94)
                     mapfix 25% (93)
                      anvil 25% (92)
                     xdecor 24% (90)
                  mob_horse 24% (89)
             3d_armor_sfinv 24% (89)
             chat_anticurse 23% (84)
                   maptools 22% (83)
                   moreores 22% (83)
               simple_skins 22% (82)
              technic_armor 21% (80)
                   throwing 20% (75)
                3d_armor_ip 20% (75)
             basic_machines 20% (73)
                  no_guests 19% (72)
                      spawn 19% (72)
                   mesecons 19% (72)
                   easyvend 19% (70)
                  anticheat 18% (69)
             homedecor_i18n 18% (66)
                     server 17% (62)
                     soccer 17% (62)
                 caverealms 16% (60)
                    intllib 16% (60)
```

**Comment by dsohler** [comment]
> https://pastebin.com/raw/2Hab0W1V

Now take the top ten mods (sans server-specific mods) and we have a list of actually used mods that could be featured.
